conda_recipes/caffe2/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
```

Comment:
So now we have the main ```build.sh``` and these specific ```build.sh``` for different conda targets.

In the same spirit of avoiding copy-pasta, let's please restructure things such that the main ```build.sh``` uses those. This is clearly a outside of the scope of this PR and we can do it in a later.

Reply:
conda packaging env variables are different. Each package needs to be independent enough to build. I can explain to you in-person how conda packaging works, that might be higher bandwidth and will help answer this question better. 

also, I believe you wanted to remove the main build.sh as well. is that still your goal?

Reply:
opened #78

Reply:
closed and responded! thanks!